app/src/components/main/modal/modalLayout/modalLayout.jsx

```diff
@@ -100,20 +92,20 @@ export class ModalLayout extends Component {
       closeConfirmed,
     });
   };
-  closeModal = () => {
+  closeModal = (outsideClick) => {
     const { closeConfirmation } = this.props;
 
     if (closeConfirmation) {
-      this.closeModalWithConfirmation();
+      this.closeModalWithConfirmation(outsideClick);
     } else {
       this.setState({ shown: false });
     }
   };
-  showCloseConfirmation = () => {
+  showCloseConfirmation = (outsideClick) => {
     const { closeConfirmed } = this.state;
     const { closeConfirmedCallback, withCheckbox } = this.props.closeConfirmation;
 
-    if (closeConfirmed) {
+    if (closeConfirmed && !(outsideClick && outsideClick.outsideClick)) {
```

Comment:
Could you clarify what happens here? Why do we need this `outsideClick`?

Reply:
I see. I think it's better to get back to the previous solution in `onClickModal` ( https://github.com/reportportal/service-ui/pull/920/files/df28416ffaa8a374eb752565ea01bb7b80288156..ba9b95014e1cc3561677859a1603b4be22148144#diff-25c069785ff0f58b8d29e94a6a0b0cf8L85 )
Sorry for misleading.

Reply:
Ok. Done